etc/jshintrc-sync-function-template.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This file is copied by the `prepare-tests.sh` script to the project's `build/sync-functions` directory to ensure the JSHint rules are applied to generated sync functions.